### DISCUSSION TITLE
Installed on Docker Desktop, getting login screen but cannot go further

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [Cloud-CV/EvalAI#3792] [Q&A]
I installed docker desktop and compose.  All seems to be fine up until login.  Tried all three usernames and create a new account AOK.

But when hitting the Login button, nothing happens, no error, nothing. Terminal also shows nothing.

Running terminal in VSCode in the evalai folder where I did the docker compose build

I get a overlay image thank you but it always returns to login

**Comment by artnuke** [comment]
Same problem. 
Does anyone know how to fix it?

**Comment by niteshsiingh** [comment]
Can you help me in the setup using docker?
The build is continuously showing errors again and again
